demos/api/stories/api.stories.tsx

```diff
@@ -9,7 +9,7 @@ stories.add('Drawer', docFn, { notes: { markdown: getReadMe('Drawer.md') } });
 stories.add('Empty State', docFn, { notes: { markdown: getReadMe('EmptyState.md') } });
 stories.add('Header', docFn, { notes: { markdown: getReadMe('Header.md') } });
 stories.add('Hero', docFn, { notes: { markdown: getReadMe('Hero.md') } });
-stories.add('Icon Wrapper', docFn, { notes: { markdown: getReadMe('IconWrapper.md') } });
+stories.add('Icons', docFn, { notes: { markdown: getReadMe('Icons.md') } });
```

Comment:
You can remove the Icon Usage entry in the welcome section (last line of welcome.stories.tsx) since it is a duplicate now.